views/topic/topic.hbs

```diff
@@ -167,21 +162,16 @@
                         {{#userHasPermission "HOMEWORK_CREATE"}}
                             <a href="/homework/new?course={{../courseId}}&topic={{../_id}}&private=true"
                                 class="btn btn-secondary btn-sm pull-right ml-1" data-toggle="tooltip"
-                                title="{{$t "global.button.addPrivateTask"}}">
-                                <i class="fa fa-plus" aria-hidden="true"></i><span class="hidden-md-down"> {{$t "global.button.addPrivateTask" }}</span>
+                                title="{{$t "global.button.addDraftTask"}}">
+                                <i class="fa fa-plus" aria-hidden="true"></i><span class="hidden-md-down"> {{$t "global.button.addDraftTask" }}</span>
                             </a>
                         {{/userHasPermission}}
                         <a data-toggle="collapse" class="collapse-toggle" href="#myhomeworks" tabindex="0">
                             <h2 class="h4">
                                 <small data-toggle="tooltip" title="{{$t "topic._topic.label.expandPrivateTasks" }}"><i
                                         class="fa fa-chevron-right collapse-icon" aria-hidden="true"></i></small>
-                                {{$t "topic._topic.headline.privateTasks" }}
-                                <small>({{arrayLength myhomeworks}}
-                                    {{#ifneq 1 (arrayLength myhomeworks)}}
-                                        {{$t "global.headline.tasks"}}
-                                    {{else}}
-                                        {{$t "global.headline.task"}}
-                                    {{/ifneq}})
+                                {{$t "topic._topic.headline.draftTasks" }}
+                                <small>({{arrayLength myhomeworks}})
```

Comment:
Same here, we're deleting the headling?

Reply:
Private tasks (0 task) vs Private tasks (0)
From a UX point of view, it is sufficient to only specify the number. Please also consider a mobile view.